packml_msgs/msg/Stats.msg

```diff
@@ -0,0 +1,29 @@
+# This message contains machine agnostic information that is extracted from PackML
+# standard states
+
+std_msgs/Header stamp       # Duration of statistics calculation
+std_msgs/Duration duration      # Duration over which statistics are calculated
```

Comment:
Note that `Duration` is a `(seconds; nanoseconds)` pair in the ROS msg spec, while PackML's `Time` is a signed integer counting only seconds.